apps.yml

```
@@ -2031,6 +2031,7 @@ apps:
     - hris_is_staff
     - team_moco
     - team_mofo
+    - non-moco-sheriffs-basic
```

To include [this `mozilliansorg` group](https://people.mozilla.org/a/non-moco-sheriffs-basic/) you'll need to grant access to `mozilliansorg_non-moco-sheriffs-basic`
```suggestion
    - mozilliansorg_non-moco-sheriffs-basic
```